CHANGES.md

```diff
@@ -4,7 +4,11 @@
 # 2.0.1 (2018-01-11)
 
 - [NEW] Changed to use nodejs-cloudant for database requests.
+- [NEW] Added API for upcoming IBM Cloud Identity and Access Management support
+  for Cloudant on IBM Cloud. Note: IAM API key support is not yet enabled in the
+  service.
 - [IMPROVED] Added compression to restore process requests.
+- [IMPROVED] Added some URL validation.
```

Comment:
Instead of using 'some', maybe it could say something like "Added Cloudant URL and IAM API key validation".

Reply:
I updated this in 7680793533fcfa2c4d45d7256175a43eddde8537
`- [IMPROVED] Added URL validation for presence of host and database elements.`
I think this makes sense because that is the improvement from the previous version. The validation of the IAM API key itself is part of adding the support for IAM. Let me know if you agree.